test/export/test_torchbind.py

```diff
@@ -1,4 +1,5 @@
 # Owner(s): ["oncall: export"]
+# ruff: noqa: F841
```

Comment:
duplicate noqa?

Reply:
My guess is that having this top level one  makes the ones below redundant?
To keep things easy to read, let's have either a full file disable, or inline ones, but not both.